dspace/config/spring/api/crosswalks.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:util="http://www.springframework.org/schema/util"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd" >
+
+    <bean name="mapConverterOpenAccessScopus" class="org.dspace.util.SimpleMapConverter" init-method="init">
```

Comment:
Why is this separate from the other Scopus configs? Is there a way to move it over with the other Scopus configs to make it easier to find?

Reply:
Feedback here wasn't answered or addressed. It's unclear why this bean is separate from all other Scopus beans.  Ideally they should all be in the same Spring XML file to ensure they are easy to configure together.

Reply:
Feedback here wasn't answered or addressed. It's unclear why this bean is separate from all other Scopus beans. Ideally they should all be in the same Spring XML file to ensure they are easy to configure together.